test/external/external_test.go

```diff
@@ -75,13 +84,35 @@ var _ = Describe("External tests", func() {
 	})
 
 	Scenario("running chart move to intermediate bundle", func() {
+		start := time.Now()
+		steps.When("clear relok8s-save-cache")
 		steps.When(fmt.Sprintf("running relok8s chart move -y ../fixtures/testchart --image-patterns ../fixtures/testchart.images.yaml --to-intermediate-bundle %s/testchart-intermediate.tar", tmpDir))
 		steps.And("the move is computed")
 		steps.Then("the command says it will archive the chart")
 		steps.Then("the command says it is writing the hints file")
 		steps.Then("the command says it is writing the Helm Chart files")
 		steps.Then("the command says it is writing the container images")
 		steps.Then("the command says the intermediate bundle is complete")
+		steps.Then("relok8s-save-cache contains expected layer")
+		firstSaveDuration := time.Since(start)
+
+		start2 := time.Now()
+		info, err := os.Stat(expectedLayerFile())
+		Expect(err).ToNot(HaveOccurred())
+		modtime := info.ModTime()
+		steps.When(fmt.Sprintf("running relok8s chart move -y ../fixtures/testchart --image-patterns ../fixtures/testchart.images.yaml --to-intermediate-bundle %s/testchart-intermediate-2.tar", tmpDir))
+		steps.And("the move is computed")
+		steps.Then("the command says it will archive the chart")
```

Comment:
Is there much value on again checking the output of the command? Could we just check on the second run that it does not expose any error and that the layer is there?

Reply:
Can remove

Reply:
Done